ts/graphs/CardCounts.svelte

```diff
@@ -111,6 +111,7 @@ License: GNU AGPL, version 3 or later; http://www.gnu.org/licenses/agpl.html
 
     .right {
         text-align: right;
+        padding-left: 40px;
```

Comment:
Maybe this could be more cleanly done by styling the table instead of a particular class - the CSS gurus here may have a suggestion for you.

One issue we need to consider is how this will appear or a narrow Phone screen. Here's what it looks like after this change:
<img width="344" alt="Screen Shot 2021-12-24 at 9 38 37 am" src="https://user-images.githubusercontent.com/180542/147300130-0e5eaad5-68ec-4168-9c2d-eedd5aabb088.png">

And before (which was not ideal either):

<img width="337" alt="Screen Shot 2021-12-24 at 9 39 39 am" src="https://user-images.githubusercontent.com/180542/147300160-96f6a3aa-a2c1-400d-acb9-5f0c497c0bad.png">

Bearing in mind that some languages can be up to 2x longer than English, I think we probably need to reduce the padding from 40 to something smaller, or use styling that is dependent on the width of the screen.

Reply:
`./script/ts-run` was very helpful, thanks! Is there a way to also have it hot-reload, so we don't have to restart anki every time?

Using `padding-left: 4vw` seems to work pretty nicely from desktop to iPhone SE (attached). Ends up like an average of the two above, which is probably the best we can get with this layout

<img width="252" alt="shrink_screenshot_out" src="https://user-images.githubusercontent.com/29561440/147303105-3c3af684-612d-44a9-9bef-3ac3455b08f5.png">

Alternatively, we'd probably achieve something much better by just switching to `flex-direction: column` on narrow devices. Something like this, though this mockup is a bit ideal

<img width="252" alt="shrink_screenshot_out" src="https://user-images.githubusercontent.com/29561440/147303313-79e09bdf-6cf4-4f20-a5f6-b2af7a7a0071.png">

Reply:
The spacing there looks good! Switching to column might be a better long term solution, but no need to tackle that in this PR.

Running scripts/ts-watch in a separate terminal will automatically rebuild on file change, then you can just refresh the page in Chrome

Reply:
The automatic repositioning seems to work well; thanks for that.